about/hardware.md

```diff
@@ -102,4 +102,6 @@ find ~ -name `*.gpw` -delete
 
 User data stored on Arjuna is **NOT** backed up, users are responsible for backing up
```

Comment:
This technically doesn't make sense. Make it User data ... is not backed up by the administration team or something like that